src/Tickets/Emails/Email_Abstract.php

```diff
@@ -873,4 +874,23 @@ public function get_post_type_data(): array {
 
 		return $data;
 	}
+
+	/**
+	 * Get the related JSON data for this email.
+	 *
+	 * @since TBD
+	 *
+	 * @return array
+	 */
+	public function get_json_data(): array {
+		/**
+		 * Allow filtering the JSON data for this email.
+		 *
+		 * @since TBD
+		 *
+		 * @param array $data The JSON data.
+		 * @param Email_Abstract $this The email object.
+		 */
+		return apply_filters( 'tec_tickets_emails_json_data', Handler::get_data( $this ), $this );
+	}
```

Comment:
Allow the filtering based on slug, like `tec_tickets_emails_{$email_slug}_json_data`